src/apps/adop_train.cpp

```diff
@@ -307,6 +307,7 @@ class NeuralTrainer
         torch::set_num_threads(1);
 
         std::string experiment_name = Saiga::CurrentTimeString("%F_%T") + "_" + params->train_params.name;
+        std::replace(s.begin(), s.end(), ':', '_'); // files with : forbiden in some linux distributives
```

Comment:
Have you compiled this change? What is `s` here?

Reply:
Sorry, I didn't. Fixed it.